src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +17,7 @@
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    List<Machine> getAll(Class<? extends T> type);
```

Comment:
```suggestion
    List<T> getAll(Class<? extends T> type);
```
тут краще поветати T замість Machine, адже ти обмежуєш вхідний парметр `? extends T` тобто ти очікуєш що буде повернуто все що стоїть нижче по ієрархії від Т, а Machine стоїть вище поієрархії